maps/away/ships/freebooter/freebooter_salvager/freebooter_salvager.dm

```diff
@@ -0,0 +1,150 @@
+/datum/map_template/ruin/away_site/freebooter_salvager
+	name = "Freebooter Salvager Ship"
+	id = "freebooter_salvager_ship"
+	description = "A heavily modified freighter of dubious origins."
+
+	prefix = "ships/freebooter/freebooter_salvager/"
+	suffix = "freebooter_salvager.dmm"
+
+	traits = list(
+		list(ZTRAIT_AWAY = TRUE, ZTRAIT_UP = TRUE, ZTRAIT_DOWN = FALSE),
+		list(ZTRAIT_AWAY = TRUE, ZTRAIT_UP = FALSE, ZTRAIT_DOWN = TRUE),
+	)
+
+	ship_cost = 1
+	spawn_weight = 1
+
+	shuttles_to_initialise = list(/datum/shuttle/autodock/overmap/freebooter_salvager, /datum/shuttle/autodock/multi/lift/fbs)
+	sectors = list(ALL_POSSIBLE_SECTORS)
+	ban_ruins = list(/datum/map_template/ruin/away_site/freebooter_ship)
+
+	unit_test_groups = list(1)
+
+/singleton/submap_archetype/freebooter_salvager
+	map = "Freebooter Salvager Ship"
+	descriptor = "A heavily modified freighter of dubious origins."
+
+/obj/effect/overmap/visitable/ship/freebooter_salvager
+	name = "Freebooter Salvager Ship"
+	desc = "Einstein Engines-manufactured Tartarus-class Bulk Haulers were a common sight in both well-charted and poorly-charted regions between systems. By design, they were made to endure most kinds of demanding trips, thanks to their thick hull and side thrusters, which decently shield the main section. However, the development of newer freighters left this class in the dust in terms of fuel efficiency and navigational support, putting it on the verge of becoming obsolete. It is not uncommon to see this model modified with improvised weaponry and other alterations in the hands of dubious actors."
+	class = "ICV"
+	icon_state = "tramp"
+	moving_state = "tramp_moving"
+	color = "#a0a8ec"
+	max_speed = 1/(2 SECONDS)
+	burn_delay = 1 SECONDS
+	vessel_mass = 10000 // very inefficient in terms of thrust
+	fore_dir = SOUTH
+	vessel_size = SHIP_SIZE_SMALL
+	invisible_until_ghostrole_spawn = TRUE
+	designer = "Einstein Engines"
+	volume = "56 meters length, 39 meters beam/width, 22 meters vertical height"
+	drive = "Low-Speed Warp Acceleration FTL Drive"
+	weapons = "Dual improvised weapon arrays, underside shuttle docking compartment."
+	sizeclass = "Tartarus-class Bulk Hauler"
+	shiptype = "Long-term shipping utilities"
+	initial_restricted_waypoints = list(
+		"Freebooter Salvager Shuttle" = list("fbs_nav_hangar")
+	)
+	initial_generic_waypoints = list(
+		"fbs_nav1",
+		"fbs_nav2",
+		"fbs_nav3",
+		"fbs_nav4",
+		"fbs_nav5",
+		"fbs_nav6",
+		"fbs_nav7",
+		"fbs_nav8",
+		"fbs_upfore"
```

Comment:
all of these should have descriptive tags and type paths
nobody is going to know what `fbs` means when debugging stuff